src/containers/Tenant/Tenant.tsx

```diff
@@ -28,8 +28,8 @@ import './Tenant.scss';
 const b = cn('tenant-page');
 
 export const TABLE_TYPE = 'Table';
-export const OLAP_TABLE_TYPE = 'OlapTable';
-export const OLAP_STORE_TYPE = 'OlapStore';
+export const OLAP_TABLE_TYPE = 'ColumnTable';
+export const OLAP_STORE_TYPE = 'ColumnStore';
```

Comment:
Maybe we can use enum here?

Reply:
This just fixes the bug. I don't want to invest more time untangling this code rn.

Reply:
ok, got it!